pkg/client/didexchange/client_test.go

```diff
@@ -387,3 +387,49 @@ func TestService_MsgEvents(t *testing.T) {
 	err = c.UnregisterMsgEvent(ch1)
 	require.NoError(t, err)
 }
+
+func TestTestEventProperties(t *testing.T) {
+	// no service error
+	ev, svcErr, err := EventProperties(&mockDIDExchangeEvent{})
+	require.NoError(t, err)
+	require.Nil(t, svcErr)
+	require.NotNil(t, ev.ConnectionID())
+	require.NotNil(t, ev.InvitationID())
+
+	// service error
+	ev, svcErr, err = EventProperties(&mockDIDExchangeEventError{})
+	require.NoError(t, err)
+	require.NotNil(t, svcErr)
```

Comment:
I don't like it `require.NotNil(t, svcErr)` 
I would rather use `Contains or EqualError`  functions and check the specific error message. It will be clear to understand what you expect here and it will prevent misunderstanding after some changes in the future.

Reply:
I missed to add it. Added in [new patch](https://github.com/hyperledger/aries-framework-go/pull/480/files#diff-415ff83c536dcba8813f0209c1104a1dR399).